src/test/benchmark.ts

```diff
@@ -0,0 +1,153 @@
+import faker from 'faker';
+import { VNode } from '../vnode';
+import h from '../h';
+import { init as curInit } from '../snabbdom';
+import { init as refInit } from 'latest-snabbdom-release';
+import { assert } from 'chai';
+import 'core-js/stable/array/fill';
+import pReduce from 'p-reduce';
+import pMapSeries from 'p-map-series';
+import { std, mean } from 'mathjs';
+
+const RUNS = 5;
+const PATCHES_PER_RUN = 500;
+const WARM_UP_RUNS = 1;
+const REQUEST_ANIMATION_FRAME_EVERY_N_PATCHES = 1;
+const BENCHMARK_TIMEOUT_MINUTES = 10;
+
+declare global {
+  const __karma__: {
+    info(info: unknown): void
+  };
+}
+
+describe('Benchmark', () => {
+  it.only('Core', async function Benchmark () {
```

Comment:
Notice this `only` call here. So to run the benchmark:

```
$ npm run compile-es -- --watch
```

And in another shell:

```
$ npm run unit:es -- --no-single-run
```